src/gas_estimator.rs

```diff
@@ -5,6 +5,13 @@ use clarity::Uint256;
 use std::cmp::Ordering;
 use std::collections::VecDeque;
 use std::time::Instant;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum GasTrackerError {
```

Comment:
implementing error is extremely straightforward, see the example in web30's error.rs file, so lets not use the crate thiserror

Reply:
I like thiserror because it macro expands to a regular error, just with less boilerplate. I removed it anyway because I think the error was unnecessary, but I think we could use it in the future for some other errors. I'm not sure we could put it in [dev.dependencies] but I would like to if we use it.